### PR TITLE
EE-2244-correct - Added JSON Assay/Experiment examples

### PR DESCRIPTION
**Pull request by M-casado**
Pull request of the correct EE-2244 ticket. 

- [x] Few **amends** to the common definitions. 
- [x] A basic **ArrayExperiment** JSON document added. 
- [x] A basic **ArrayAssay** JSON document added. 

Both of these JSON documents (with what I would consider minimum/common information) were **based on** the **EGAS00001004087** Array study from EGA, picked by @colthom as a normal Array Submission, and its [**article**](https://www.life-science-alliance.org/content/3/7/e202000650) (because the spreadsheet lacked information).

Please note three things:
- **These two JSON documents do not account for a full submission** (we would need samples, dataset, analyses, study...).  
- The **ArrayAssay is expected to hold the raw data**, not the processed files. These (RData and CSV files from the same spreadsheet) would be submitted as **analyses**. 
- The ArrayAssay holds information of a single Assay from a single sample. Within the ArrayExperiment **there would be another 123 ArrayAssays** (to a total of 124 samples-assays), which would be similar to the one added in this PR.

### CODE REVIEW

**Comment by M-casado** [comment]
@sdelatorrep and @colthom, since this branch ``EE-2244-correct`` was created from the previous branch ``EE-2244`` that is still pending review (but had the latest changes), their commits seem to appear within this PR. But this PR own changes are the ones beyond ``EE-2244 - Small documentation amend`` ([046e9be](https://github.com/EbiEga/ega-metadata-schema/pull/28/commits/046e9bee74b194060f0b7149e584d9413a6fa7d6))

File: docs/json_browser/markdowns/ega-1-properties-array-type-of-the-arrayexperiment-efo0002696.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Type of array (not its purpose per se) providing a glimpse of the used technolog
...
+| Property                              | Type     | Required | Nullable       | Defined by                                                                                                                                                                                                                                                                                                        |
+| :------------------------------------ | :------- | :------- | :------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [array_type_id](#array_type_id)       | `string` | Required | cannot be null | [EGA ArrayExperiment metadata schema v0.0.1](ega-1-properties-array-type-of-the-arrayexperiment-efo0002696-properties-array-type-of-the-arrayexperiment---id.md "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.ArrayExperiment.json#/properties/array_type/properties/array_type_id")       |
```
Comment (afoix) [inline docs/json_browser/markdowns/ega-1-properties-array-type-of-the-arrayexperiment-efo0002696.md:21]: Question: is the change here EGA introducing versioning in the name of the schema? (nothing important just curiosity)

File: docs/json_browser/markdowns/ega-2-definitions-check-that-the-object_ids-accession-pattern-and-object_type-match-anyof-individual-object_id-and-object_type-check-properties-object_id-properties-pattern-of-an-ega-individuals-id-egai.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## ega_accession Type
+
+`string` ([Pattern of an EGA Individual's ID (EGAI...)](ega-2-definitions-check-that-the-object_ids-accession-pattern-and-object_type-match-anyof-individual-object_id-and-object_type-check-properties-object_id-properties-pattern-of-an-ega-individuals-id-egai.md))
+
+## ega_accession Constraints
+
+**pattern**: the string must match the following regular expression: 
+
+```regexp
+^EGAI[0-9]{11}$
```
Comment (afoix) [inline docs/json_browser/markdowns/ega-2-definitions-check-that-the-object_ids-accession-pattern-and-object_type-match-anyof-individual-object_id-and-object_type-check-properties-object_id-properties-pattern-of-an-ega-individuals-id-egai.md:22]: Validated with [Regex101](https://regex101.com/) . The pattern is correct and matches the EGAI 😊

**Review by afoix — APPROVED** [review]
Looks good to me! 🥳

**Pull request merged**